invenio_github/views/github.py

```diff
@@ -146,8 +151,11 @@ def get_repository(repo_name):
             )
         except RepositoryAccessError:
             abort(403)
-        except NoResultFound:
+        except (NoResultFound, RepositoryNotFoundError):
             abort(404)
+        except Exception as exc:
+            current_app.logger.exception(str(exc))
+            abort(400)
```

Comment:
We discussed about adding error handlers that could use our default error handling. That way, we could benefit from sending the sentry error ID in the response and that could help us debug issues faster.

However, that means we need to change a bit the code here. 

In order not to block the fix for shared repos, I will tackle that [separately](https://github.com/inveniosoftware/invenio-github/issues/156).